### ISSUE TITLE
Expand "using local env vars" article

### ISSUE THREAD

**Issue by Tonnu** [wercker/docs#315]
It should include some more information on how to set public and protected env vars with the `X_` and `XXX_` prefixes respectively.

**Comment by hatchan** [comment]
There's already an article about using local env var with `X_`: http://devcenter.wercker.com/docs/using-the-cli/local-env-vars.html

And there is no need to explain the `XXX_` as this is just for the hosted situation.

**Comment by Tonnu** [comment]
Makes sense for the `XXX_` prefix, but mentioning the `ENVIRONMENT` file should be in there though.
